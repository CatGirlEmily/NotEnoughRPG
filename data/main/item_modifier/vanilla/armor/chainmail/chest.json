[
  {
    "function": "minecraft:set_components",
    "components": {
      "minecraft:custom_data": "{CHAINMAIL_BOOTS:true,rarity:true}"
    },
    "conditions": []
  },
  {
    "function": "minecraft:set_lore",
    "entity": "this",
    "lore": [
      "§7Defense: §a+30",
      " ",
      "§a§lUNCOMMON CHESTPLATE"
    ],
    "mode": "replace_all"
  }
]