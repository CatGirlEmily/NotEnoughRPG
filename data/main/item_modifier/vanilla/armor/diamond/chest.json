[
  {
    "function": "minecraft:set_components",
    "components": {

      "minecraft:custom_data": "{DIAMOND_CHESTPLATE:true,rarity:true}"
    },
    "conditions": []
  },
  {
    "function": "minecraft:set_lore",
    "entity": "this",
    "lore": [
      "§7Defense: §a+40",
      " ",
      "§a§lUNCOMMON CHESTPLATE"
    ],
    "mode": "replace_all"
  }
]