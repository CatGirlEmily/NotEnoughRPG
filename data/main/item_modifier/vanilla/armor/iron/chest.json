[
  {
    "function": "minecraft:set_components",
    "components": {

      "minecraft:custom_data": "{IRON_CHESTPLATE:true,rarity:true}"
    },
    "conditions": []
  },
  {
    "function": "minecraft:set_lore",
    "entity": "this",
    "lore": [
      "§7Defense: §a+30",
      " ",
      "§f§lCOMMON CHESTPLATE"
    ],
    "mode": "replace_all"
  }
]