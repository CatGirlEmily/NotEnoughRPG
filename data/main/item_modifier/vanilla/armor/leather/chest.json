[
  {
    "function": "minecraft:set_components",
    "components": {

      "minecraft:custom_data": "{LEATHER_CHESTPLATE:true,rarity:true}"
    },
    "conditions": []
  },
  {
    "function": "minecraft:set_lore",
    "entity": "this",
    "lore": [
      "§7Defense: §a+15",
      " ",
      "§f§lCOMMON CHESTPLATE"
    ],
    "mode": "replace_all"
  }
]