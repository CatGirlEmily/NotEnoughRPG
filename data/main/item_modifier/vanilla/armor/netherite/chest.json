[
  {
    "function": "minecraft:set_components",
    "components": {

      "minecraft:custom_data": "{NETHERITE_CHESTPLATE:true,rarity:true}"
    },
    "conditions": []
  },
  {
    "function": "minecraft:set_lore",
    "entity": "this",
    "lore": [
      "§7Defense: §a+50",
      " ",
      "§5§lEPIC CHESTPLATE"
    ],
    "mode": "replace_all"
  }
]