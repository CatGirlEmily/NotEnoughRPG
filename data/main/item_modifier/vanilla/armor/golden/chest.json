[
  {
    "function": "minecraft:set_components",
    "components": {

      "minecraft:custom_data": "{GOLDEN_CHESTPLATE:true,rarity:true}"
    },
    "conditions": []
  },
  {
    "function": "minecraft:set_lore",
    "entity": "this",
    "lore": [
      "§7Defense: §a+25",
      " ",
      "§f§lCOMMON CHESTPLATE"
    ],
    "mode": "replace_all"
  }
]